scripts/jaseci.yaml

```diff
@@ -17,11 +17,12 @@ metadata:
   namespace: default
 data:
   prod_up: |-
-    apt update; apt -y upgrade; apt -y install --no-install-recommends git g++;
+    apt update; apt -y upgrade; apt -y install --no-install-recommends git g++ pkg-config cmake build-essential;
```

Comment:
Why are these new dependences needed here?

Reply:
Otherwise jaseci kit installer fails.

Reply:
okay i see. Yea let's not commit this to the main repo and just keep this as a separate jaseci yaml file that we use for our experiments.

Reply:
So i just create another file in script folder and then do the same thing?

Reply:
Yes and lets not commit that to the main jaseci repo. you can just leave it as a unstaged file on your machine.

Reply:
Ok